Server/JokesOnYou.Web.Api/Controllers/AuthController.cs

```diff
@@ -7,16 +10,53 @@
 
 namespace JokesOnYou.Web.Api.Controllers
 {
-    public class AuthController
+    [Route("api/[controller]")]
+    [ApiController]
+    public class AuthController : ControllerBase
     {
-        public ActionResult Login([FromBody] UserLoginDTO userLoginDto)
+        private readonly IAuthService _authService;
+
+        public AuthController(IAuthService authService)
         {
-            throw new NotImplementedException();
+            _authService = authService;
         }
 
-        public Task<ActionResult> Register([FromBody] UserRegisterDTO userRegisterDto)
+        [AllowAnonymous]
+        [HttpPost("login")]
+        public async Task<IActionResult> Login(UserLoginDTO userLoginDto)
         {
-            throw new NotImplementedException();
+            try
+            {
+                var user = await _authService.LoginAsync(userLoginDto);
+                return Ok(user);
+            }
+            catch (UserLoginException ex)
```

Comment:
we should setup the global exception handler, so good idea to save possible exceptions.

Reply:
Yes we should I need to read up on global exception handlers ;) though this is not an unhandled exception so this should not give an issue and I think is good practice.